lib/mws/cart_information/client.rb

```diff
@@ -16,7 +16,8 @@ module CartInformation
     # @note In addition to registering for Amazon MWS, you must also request
     #   authorization to use the Cart Information API section.
     class Client < ::Peddler::Client
-      path '/CartInformation/2014-03-01'
+      version '2014-03-01'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.